packages/components/src/Slider/Slider.component.js

```diff
@@ -26,6 +26,9 @@ function isIconsAvailables(icons) {
  * @param {number} max - maximum value of the slider
  */
 export function getSelectedIconPosition(icons, value, min, max) {
+	if (value === null) {
+		return -1;
```

Comment:
why not throwing and forcing the client of this function to handle the error case cleanly ?

Reply:
It's not an "error", it allow us to reset the slider to "no value" state.